test/functionalTest/cases/2750_common_metrics/2750_incoming_errors.test

```diff
@@ -295,25 +302,44 @@ Date: REGEX(.*)
                     "serviceTime": REGEX(.*)
                 },
                 "SP/get/metrics": {
-                    "incomingTransactionResponseSize": 337,
+                    "incomingTransactionResponseSize": 457,
```

Comment:
Remember the `fixContentLengths.py` script can be used to automatically adjust Content-Length (when the only difference between .test and .out is Content-Lenght lines). Your nightmare can become in a sweet dream xD